test.py

```diff
@@ -1,4 +1,8 @@
 from test1 import func
 
 print("my")
-print("world")
\ No newline at end of file
+print("test1")
+print("testtttt")
+
+def func():
```

Comment:
Added a new function for a more complex test case.
```suggestion
|
```
N/A